tests/unish/UnishIntegrationTestCase.php

```diff
@@ -133,7 +133,7 @@ protected function buildCommandLine($command, $args, $options)
         }
         // insert drush command options
         foreach ($options as $key => $value) {
-            if (!isset($value)) {
+            if (!isset($value) || $value === true) {
```

Comment:
Usually, `'yes' => true` gives `--yes`, and `'yes' => '1'` gives `--yes=1`. I've always found the Unish-style `'yes' => null` to be confusing. Would it be okay to switch to `true` everywhere?  We should also support `null` for external unish clients.

Reply:
Yes feel free to change that.